lib/panolint/version.rb

```diff
@@ -1,5 +1,5 @@
 # frozen_string_literal: true
 
 module Panolint
-  VERSION = "0.1.2"
+  VERSION = "0.1.3"
```

Comment:
From Jacob:
> We actually never push it to RubyGems (Kevin did for the first 2 versions IIRC but then we decided we could just use commit hashes in the Gemfiles instead).

So I don't think the version push is needed. That being said, I don't think it would hurt either? So 🤷

Reply:
@parkerfinch lmk what I should opt for here!